content/articles/2019-04-dbplyr-1-4-0.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: dbplyr 1.4.0
+author: Hadley Wickham
+date: '2019-04-30'
+slug: dbplyr-1-4-0
+categories:
+  - package
+tags:
+  - databases
+photo:
+  url: https://unsplash.com/photos/JuFcQxgCXwA
+  author: Samuel Zeller
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're stoked to announce the release of dbplyr 1.4.0, the database backend for dplyr that translates R code into SQL. dbplyr allows you to use a remote database as if it was a collection of local data frames: you write ordinary dplyr code and dbplyr turns it into SQL for you.
+
+You can install the released version from CRAN:
+
+```{r, eval = FALSE}
+install.packages("dbplyr")
+```
+
+While there are a large number of small improvements and bug fixes (see the full full details in the [changelog](https://dbplyr.tidyverse.org/news/index.html)), the biggest improvements in this release are related to SQL generation. In this blog post, I'll show you how find out what dplyr is doing behind the scenes, discuss some major simplifications to the SQL that dbplyr generates, and then show off a few of the new function translations.
+
+To get started, I'll load dplyr and dbplyr:
+
+```{r setup, message = FALSE}
+library(dplyr)
+library(dbplyr)
+```
+
+## SQL simulation
+
+This version of dbplyr substantially improves the tools that allow you to see what SQL dbplyr generates without needing to connect to a live database. You won't generally need these tools for real data, but they're very useful for blog posts (like this one!), for for generating reprexes (as discussed in the new [`vignette("reprex")`](https://dbplyr.tidyverse.org/articles/reprex.html)), and dbplyr's internal tests. 
```

Comment:
* and **for** dbplyr's internal tests (list agreement, since you have for blog posts, for generating reprexes
* remove duplicate for in "for for generating reprexes"